src/brain/issueLabelHandler/route.ts

```diff
@@ -226,9 +216,19 @@ export async function markRouted({
     owner: owner,
     repo: payload.repository.name,
     issue_number: payload.issue.number,
-    labels: [UNTRIAGED_LABEL, WAITING_FOR_PRODUCT_OWNER_LABEL],
+    labels: [UNTRIAGED_LABEL],
   });
 
+  // Only retriage issues if support is routing
+  if (labelNames.includes(WAITING_FOR_SUPPORT_LABEL)) {
```

Comment:
What do you think about this, `Waiting for: *` label only changes is support is doing the routing?

Reply:
I like it! This way if some other EPD member at Sentry updates the product area then it will not automatically go to triage, but we can assume Support does mean to retriage. 👍

If you want to use the other suggestion about, then also this:

```suggestion
  if (isBeingRoutedBySupport) {
```